### PR TITLE
GFO-48: Displaying tickets with their QR codes after downloading them from backend and adding into store

### PR DESCRIPTION
**Pull request by tamas-w**

### CODE REVIEW

File: frontend/src/components/UserTicketList/UserTicketList.jsx [hunk 1]
```diff
@@ -1,41 +1,62 @@
...
+  return {
+    downloadTickets: () => dispatch(getTickets),
+  }
+};
+
+UserTicketList.propTypes = {
+  getTickets: PropTypes.func.isRequired,
+};
+
+export default connect(mapStateToProps, mapDispatchToProps)(UserTicketList);
```
Comment (adamgyulavari) [inline frontend/src/components/UserTicketList/UserTicketList.jsx:70]: @tamas-w empty line

File: frontend/src/flux/reducers/ticketReducer.js [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+import { ADD_TICKET, DELETE_TICKET, GET_TICKETS, DOWNLOADING_TICKETS } from '../actions/types';
 
 const initialState = {
   tickets: [],
 };
 
 export default function (state = initialState, action) {
   switch (action.type) {
+    case DOWNLOADING_TICKETS:
+      return{
```
Comment (adamgyulavari) [inline frontend/src/flux/reducers/ticketReducer.js:10]:
```suggestion
      return {
```

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by adamgyulavari — APPROVED** [review]

File: frontend/src/components/UserTicketList/UserTicketList.jsx [hunk 3]
```diff
@@ -1,41 +1,70 @@
...
-    title: 'Triple Ticket',
-  },
-];
-
-const UserTicketList = () => {
   return (
-    <div className="ticket-list-container">
-      <div className="ticket-list">
+    <>
+      {tickets[0] &&
```
Comment (adamgyulavari) [inline frontend/src/components/UserTicketList/UserTicketList.jsx:23]:
```suggestion
      {tickets.length > 0 && 
```

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by marcianszlovak — APPROVED** [review]

**Review by marcianszlovak — APPROVED** [review]

**Review by marcianszlovak — APPROVED** [review]

**Review by marcianszlovak — APPROVED** [review]
approved!

**Review by adamgyulavari — APPROVED** [review]
sadly I was the one who requested the changes, so I have to approve :D

**Pull request merged**